commands/settings.js

```diff
@@ -0,0 +1,94 @@
+const { SlashCommandBuilder } = require('@discordjs/builders')
+const { reply, defer } = require('../helpers/interactionHelper')
+const { getGuild, saveGuild } = require('../helpers/guildData')
+const { verifyAdminAsync } = require('./admin')
+const { MessageEmbed } = require('discord.js')
+
+const setMemberNotificationChannel = async interaction => {
+  const channel = interaction.options.getChannel('channel')
+  const guildData = await getGuild(interaction.guild.id)
+  guildData.memberNotificationChannelId = channel.id
```

Comment:
The bot should check whether this channel is a voice channel and he has permissions to post there before setting it